components/list/list-item.js

```diff
@@ -137,11 +180,17 @@ class ListItem extends RtlMixin(LitElement) {
 	}
 
 	render() {
-		const illustrationSlot = html`<slot name="illustration"></slot>`;
+		const contentId = getUniqueId();
+		const checkBoxId = getUniqueId();
+		const illustrationSlot = html`
+			<input @change="${this._handleChange}" class="d2l-list-item-checkbox" type="checkbox" id="${checkBoxId}">
```

Comment:
I think this should be binding to the `checked` property of the list-item. I suspect we'll want `.checked=${this.checked}`.

Reply:
Oh, miss read this see PR 4 it fixes this issue.

Reply:
Move this discussion to: https://github.com/BrightspaceUI/core/pull/212/files#r324254933